src/EventStore.Core/Services/Transport/Grpc/RpcExceptions.cs

```diff
@@ -237,9 +237,12 @@ public static Exception PersistentSubscriptionDropped(string streamName, string
 		public static RpcException InvalidArgument<T>(T argument) =>
 			new(new Status(StatusCode.InvalidArgument, $"'{argument}' is not a valid {typeof(T)}"));
 
+		public static RpcException RequiredOption<T>(string optionName, T argument) =>
+			new(new Status(StatusCode.InvalidArgument, $" `{optionName}` is a required field of type {typeof(T)}"));
```

Comment:
Tiny thing, the status code is about arguments maybe `RequiredArgument` would be better here and in the message 
```suggestion
			new(new Status(StatusCode.InvalidArgument, $" `{name}` is a required argument of type {typeof(T)}"));
```